packages/blade/src/components/BottomSheet/__tests__/BottomSheet.web.test.tsx

```diff
@@ -0,0 +1,310 @@
+/* eslint-disable no-global-assign */
+/* eslint-disable @typescript-eslint/no-explicit-any */
+import React from 'react';
+import userEvents from '@testing-library/user-event';
+import { mockViewport } from 'jsdom-testing-mocks';
+import { within } from '@testing-library/react';
+import { BottomSheet, BottomSheetHeader, BottomSheetBody, BottomSheetFooter } from '../BottomSheet';
+import renderWithTheme from '~src/_helpers/testing/renderWithTheme.web';
+import { Text } from '~components/Typography';
+import { Button } from '~components/Button';
+import { Dropdown } from '~components/Dropdown';
+import { SelectInput } from '~components/Input/SelectInput';
+import { ActionList, ActionListItem } from '~components/ActionList';
+import { Badge } from '~components/Badge';
+import { Counter } from '~components/Counter';
+
+export const sleep = (delay = 10): Promise<number> =>
+  new Promise((resolve) => setTimeout(resolve, delay));
+
+const SingleSelectContent = (): React.ReactElement => {
+  return (
+    <ActionList>
+      <ActionListItem title="Home" value="Home" />
+      <ActionListItem title="Settings" value="settings" />
+      <ActionListItem title="Info" value="info" />
+      <ActionListItem title="Price" value="price" />
+      <ActionListItem title="Contact" value="contact" />
+      <ActionListItem title="About" value="about" />
+    </ActionList>
+  );
+};
+
+const MultiSelectContent = (): React.ReactElement => {
+  const fruites = [
+    'Apple',
+    'Apricot',
+    'Avocado',
+    'Banana',
+    'Orange',
+    'Blackberry',
+    'Blueberry',
+    'Cherry',
+    'Coconut',
+    'Cucumber',
+    'Durian',
+  ];
+
+  return (
+    <ActionList>
+      {fruites.map((fruit) => {
+        return <ActionListItem key={fruit} title={fruit} value={fruit} />;
+      })}
+    </ActionList>
+  );
+};
+
+beforeAll(() => jest.spyOn(console, 'error').mockImplementation());
+afterAll(() => jest.restoreAllMocks());
```

Comment:
Let's have this only at related test blocks

Reply:
Done.